arlo-client/src/components/AuditFlow/index.tsx

```diff
@@ -75,18 +60,19 @@ const AuditFlow: React.FC<IProps> = ({
     updateAudit()
   }, [updateAudit])
 
-  const board:
-    | IAuditBoard
-    | undefined = audit.jurisdictions[0].auditBoards.find(
-    (v: IAuditBoard) => v.id === token
-  )
+  const board: IAuditBoard | undefined = audit
+    ? audit.jurisdictions[0].auditBoards.find(
```

Comment:
Also, were you on TypeScript 3.7, this whole thing could be `const board = audit?.jurisdictions[0].auditBoards.find(v => v.id === token)`.

Reply:
@eventualbuddha Typescrypt 3.7 looks awesome! A separate refactor PR that upgrades to it would be awesome.

Reply:
For sure not part of this. Just wanted to nudge for later 😄